content/en/blog/2021/non-code-contribution.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "How to choose a SIG as a non-code Kubernetes contributor"
+linkTitle: "How to choose a SIG as a non-code Kubernetes contributor"
+date: 2021-02-21
+---
+By [Chris Short](https://twitter.com/ChrisShort)
+
+Kubernetes contributors aren't people in capes or part of some secret society. How to start committing to the GitHub repos that make up the project [is well documented](https://www.kubernetes.dev/docs/guide/), yet it remains intimidating for many.
+
+A few years ago, I spoke at an event and jokingly said, "Kubernetes is just a bunch of APIs and YAML... I'm a contributor; you don't believe me?" After that talk, someone pulled me aside and asked if I was the Kubernetes contributor. They wanted to get involved in the community. Then came the real question, "I don't know which special interest group (SIG) I would work in."
+
+The SIG you work in depends on your skills and what you want to do with your free time.
```

Comment:
or your companies time :) there should be a note here that this is the preferred route so that there is a support system in place